application/src/main/java/com/sanction/lightning/facebook/FacebookService.java

```diff
@@ -102,17 +104,30 @@ public FacebookUser getFacebookUser() {
    *
    * @return A boolean to denote that the function worked.
    */
-  public FacebookPhoto publishToFacebook(byte[] inputBytes, String fileName, String message) {
-    FacebookPhoto response;
+  public String publishToFacebook(byte[] inputBytes, String type, String fileName,
+                                  String message, String videoTitle) {
+    JsonObject response;
```

Comment:
You should declare this variable closer to when you use it (i.e. right before the `try` block on line 123).